demo/scenario/script

```diff
@@ -48,11 +48,13 @@ submit Auctioneer1 endRound
 wait "End Round 4 - Clock Phase Complete  (Hit any key to continue)"
 
 # assignment rounds ...
-submit Auctioneer1 startNextRound
 # bidding not yet implemented, add here once done and
-# uncomment below endRound (Note: currently startNextRound in assignment
+# uncomment below startNextRound and endRound (Note: last endRound in clock
 # phase immediately assigns (randomly) the slots and terminates the phase)
+
+# submit Auctioneer1 startNextRound
 # submit Auctioneer1 endRound
```

Comment:
Forgot to add to review, but would be nice to create a json file for a publishAssignemtResult tx here -- essentially a `cp Auctioneer1.endRound Auctioneer1.publishAssignmentResults` .. -- and then add here a `submit Auctioneer1 publishAssignementResults` ...

Reply:
done